packages/types-algosdk/types/index.d.ts

```diff
@@ -179,6 +179,7 @@ declare module 'algosdk' {
   export interface SignedTransaction {
     txn: Transaction;
     sig: Uint8Array;
+    msig: any;
```

Comment:
Let's not use `any`. We already have a type for this.

Reply:
Oh. I will look into it.

Reply:
Done